### PR TITLE
[FIX] install: Install odoo dependencies

### PR DESCRIPTION
**Pull request by moylop260**
Since that we are using original odoo requirements we can't cached it all

So we need install them because requirements are changing between odoo version
Fix Vauxoo/docker-odoo-image#261

### CODE REVIEW

File: travis/travis_install_nightly [hunk 1]
```diff
@@ -108,21 +108,23 @@ fi
...
+sed -i '/^python-ldap\=\=/d' ${ODOO_PATH}/requirements.txt
+# Use requests with [security] suffix to fix [Errno 111] Connection refused for old python2.7 versions.
+sed -i 's/^requests\=\=/requests[security]\=\=/g' ${ODOO_PATH}/requirements.txt
+
+pip install -q --no-binary pycparser -r ${ODOO_PATH}/requirements.txt
+
 if [ "x${WITHOUT_DEPENDENCIES}" == "x" ] ; then
     echo "Installing dependencies"
     pip install --upgrade pip
     pip install -q --no-binary pycparser -r ${HOME}/maintainer-quality-tools/requirements.txt
```
Comment (JesusZapata) [inline travis/travis_install_nightly]: @moylop260 
This line should be out the if sentence `if [ "x${WITHOUT_DEPENDENCIES}" == "x" ] ; then` ?

Comment (moylop260) [inline travis/travis_install_nightly]: `;` avoid split line

File: travis/travis_install_nightly [hunk 2]
```diff
@@ -108,21 +108,23 @@ fi
 # Workaround to force using system site packages (see https://github.com/Shippable/support/issues/241#issuecomment-57947925)
 rm -f $VIRTUAL_ENV/lib/python2.7/no-global-site-packages.txt
```
Comment (JesusZapata) [inline travis/travis_install_nightly]: @moylop260 

In the line `110` shoul be add the follow lines

```bash
# Upgrade pip 
pip install --upgrade pip
```
That lines solved the follow mistake:

```bash
pip install -q --no-binary pycparser -r ${ODOO_PATH}/requirements.txt

Usage:   
  pip install [options] <requirement specifier> [package-index-options] ...
  pip install [options] -r <requirements file> [package-index-options] ...
  pip install [options] [-e] <vcs project url> ...
  pip install [options] [-e] <local project path> ...
  pip install [options] <archive url/path> ...

no such option: --no-binary
```
Actually the version of pip is `pip 6.0.8` the upgrade version is `pip 9.0.1`

Comment (moylop260) [inline travis/travis_install_nightly]: thanks merged

**Comment by osvalr** [comment]
@moylop260  this pull got a commit after it was merged to master, am I right? it seems to be related to https://github.com/Vauxoo/maintainer-quality-tools/pull/224#discussion_r131140057

**Pull request merged**